torch2trt/torch2trt.py

```diff
@@ -534,6 +537,18 @@ def torch2trt(module,
     if output_names is None:
         output_names = default_output_names(len(outputs))
 
+    profile = builder.create_optimization_profile();
+    
+    for i, name in enumerate(input_names):
+        # use fixed optimization profile
+        profile.set_shape(
+            name,
+            tuple(inputs[i].shape),
+            tuple(inputs[i].shape),
+            tuple(inputs[i].shape)
+        )
+        config.add_optimization_profile(profile)
```

Comment:
Looks like this is for dynamic shapes? Is this necessary at the moment? I believe the current API will only support one (static) shape anyway, so I think we don't need _any_ optimization profiles (ie. we'll end up just using the given shape as before).